packages/service-library/src/servicelib/long_running_tasks/_task.py

```diff
@@ -405,9 +402,13 @@ def start_task(
     # only one unique task can be running
     if unique and tasks_manager.is_task_running(task_name):
         managed_tasks_ids = list(tasks_manager.get_task_group(task_name).keys())
-        assert len(managed_tasks_ids) == 1  # nosec
-        managed_task = tasks_manager.get_task_group(task_name)[managed_tasks_ids[0]]
-        raise TaskAlreadyRunningError(task_name=task_name, managed_task=managed_task)
+        if len(managed_tasks_ids) != 1:
+            raise RuntimeError(f"Unexpected amount of entries in {managed_tasks_ids=}")
```

Comment:
Pls make sure this exception is handled properly. Isn't there any long-running-task specific exception?

Reply:
this no longer applies